Packs/PaloAltoNetworks_PAN_OS_EDL_Management/Integrations/PaloAltoNetworks_PAN_OS_EDL_Management/PaloAltoNetworks_PAN_OS_EDL_Management.yml

```diff
@@ -28,7 +28,7 @@ configuration:
   name: document_root
   required: false
   type: 0
-description: Deprecated. Use the Palo Alto Networks PAN-OS EDL Service integration instead. This integration is still supported however, for customers with over 1000 Firewalls.
+description: Deprecated. Use the Generic Export Indicators Service integration instead. This integration is still supported however, for customers with over 1000 Firewalls.
```

Comment:
Is that intentional?

Reply:
This is another small fix. I will open a separate PR for it